src/server/_config.py

```diff
@@ -57,7 +57,7 @@
 }
 NATION_REGION = "nat"
 
-API_KEY_REQUIRED_STARTING_AT = date.fromisoformat(os.environ.get("API_REQUIRED_STARTING_AT", "2023-06-21"))
+API_KEY_REQUIRED_STARTING_AT = date.fromisoformat(os.environ.get("API_KEY_REQUIRED_STARTING_AT", "2023-06-21"))
```

Comment:
oof, nice find.  that could have ended up being fairly confusing.  we should also remove or update the line in [.env.example](https://github.com/cmu-delphi/delphi-epidata/blob/1480948c1b253ba7d5733ea5618a4351924a87a8/.env.example#L4), as well as change it in [cmu-delphi/delphi-ansible-web](https://github.com/cmu-delphi/delphi-ansible-web/blob/2ed0f71123ac223e23bf2aae521e9c7710561649/vars.yml#L118)